projects/http/src/modules/auth-http-client/auth-token-provider.ts

```diff
@@ -14,8 +14,7 @@ import {
   SkyAppRuntimeConfigParamsProvider
 } from '@skyux/config';
 
-import * as jwtDecode_ from 'jwt-decode';
-const jwtDecode = jwtDecode_;
+import jwt_decode from "jwt-decode";
```

Comment:
Could we do `jwtDecode` instead of `jwt_decode`?

Reply:
Done.